app/composite-components/productsTabClient.tsx

```diff
@@ -0,0 +1,274 @@
+import {
+  Button,
+  BUTTON_TYPE, Details, FormCheckbox, FormSelect
+} from "@capgeminiuk/dcx-react-library";
+import { isEmpty } from "lodash";
+import { useEffect, useState } from 'react';
+import { AccessibleAutocomplete, ClientOnly } from '~/components';
+import { IErrorTransformed, ILabelAndValue, ISpecies } from "~/types";
+
+type ProductTabsProps = {
+  species: ISpecies[];
+  states: ILabelAndValue[];
+  presentations: ILabelAndValue[];
+  commodityCodes: ILabelAndValue[];
+  faoCode?: string;
+  stateCode?: string;
+  presentationCode?: string;
+  commodityCode?: string;
+  errors?: IErrorTransformed;
+};
+
+export const ProductsTabClient = ({
+  species,
+  states,
+  faoCode,
+  stateCode,
+  presentationCode,
+  presentations,
+  commodityCodes,
+  commodityCode,
+  errors
+}: ProductTabsProps) => {
+  const [selectedSpeciesCode, setSelectedSpeciesCode] = useState('');
```

Comment:
I feel like you need a useReducer more then useState here

Reply:
To combine the different drop down values? Yep, I can.